app/bundles/AssetBundle/Translations/en_US/messages.ini

```
@@ -12,8 +12,10 @@ mautic.protip.assets.repurpose="Repurpose high-performing assets across differen
 mautic.protip.assets.track="Track asset performance metrics in the dashboard widgets to identify top-performing content"
 mautic.asset.asset.searchcommand.lang="lang"
 mautic.asset.asset.searchcommand.lang.description="Filters assets by a specific language code"
-mautic.asset.asset.downloads.total="Total: count"
-mautic.asset.asset.downloads.unique="Unique: count"
+mautic.asset.asset.downloads.total="Total downloads"
+mautic.asset.asset.downloads.unique="Unique downloads"
+mautic.asset.asset.downloads.total.all_time="All downloads in time since this asset was created"
+mautic.asset.asset.downloads.unique.all_time="Unlike total downloads, this counts each person only once, in all time"
```

```suggestion
mautic.asset.asset.downloads.unique.all_time="Unlike total downloads, this counts each contact only once"
```

I think this reads better, and I think we should be consistent in referring to people as contacts.